### PR TITLE
Should not flush in place for stream loop.

### PR DESCRIPTION
**Pull request by derekcollison**
Signed-off-by: Derek Collison <derek@nats.io>

/cc @nats-io/core

### CODE REVIEW

File: server/stream.go [hunk 1]
```diff
@@ -2991,7 +2991,7 @@ func (mset *stream) internalLoop() {
 					pm.o.didNotDeliver(pm.seq)
 				}
 			}
-			c.flushClients(10 * time.Millisecond)
+			c.flushClients(0)
```
Comment (kozlovic) [inline server/stream.go:2994]: I think we should also look at the sendq here: https://github.com/nats-io/nats-server/blob/bd5d5d0ff89516c8702c9a0e31c004b8da315c63/server/sendq.go#L87

**Review by kozlovic — APPROVED** [review]
LGTM but may want to check the other place where we do that.

File: server/stream.go (hunk 1 above)
Comment (derekcollison) [reply server/stream.go:2994]: Good call. Agree.

Comment (derekcollison) [reply server/stream.go:2994]: ok done.

**Comment by derekcollison** [comment]
Should I squash?

**Review by kozlovic — APPROVED** [review]
LGTM

**Pull request merged**